src/test/resources/database/books/add-books.sql

```diff
@@ -0,0 +1,4 @@
+insert into books (id, title, author, isbn, price, description, cover_image, is_deleted) values (1, '1984', 'George Orwell', '978-5-699-12014-7', 19, 'Description', 'Cover image', false);
+insert into books (id, title, author, isbn, price, description, cover_image, is_deleted) values (2, 'The Lord of The Rings', 'J. R. R. Tolkien', 'some isbn2', 29, 'Description', 'Cover image', false);
+insert into books (id, title, author, isbn, price, description, cover_image, is_deleted) values (3, 'The Adventures of Sherlock Holmes', 'Sir Arthur Conan Doyle', 'some isbn3', 24, 'Description', 'Cover image', false);
+insert into books (id, title, author, isbn, price, description, cover_image, is_deleted) values (4, 'A Game of Thrones', ' George RR Martin', 'some isbn4', 34, 'Description', 'Cover image', false);
```

Comment:
U can make it by one insert
`insert into books (id, title, author, isbn, price, description, cover_image, is_deleted) values 
(1, '1984', 'George Orwell', '978-5-699-12014-7', 19, 'Description', 'Cover image', false), 
(2, 'The Lord of The Rings', 'J. R. R. Tolkien', 'some isbn2', 29, 'Description', 'Cover image', false),
(3, 'The Adventures of Sherlock Holmes', 'Sir Arthur Conan Doyle', 'some isbn3', 24, 'Description', 'Cover image', false),
(4, 'A Game of Thrones', ' George RR Martin', 'some isbn4', 34, 'Description', 'Cover image', false);
`